core/src/main/kotlin/com/katanox/tabour/sqs/config/SqsConsumerConfiguration.kt

```diff
@@ -0,0 +1,63 @@
+package com.katanox.tabour.sqs.config
+
+import com.katanox.tabour.consumption.Config
+import java.time.Duration
+import java.time.temporal.ChronoUnit
+
+class SqsConsumerConfiguration internal constructor() : Config {
+
+    /** The number of max messages to fetch. Default is 1 with max being 10 */
+    var maxMessages: Int = 1
+        @Throws(IllegalArgumentException::class)
+        set(value) {
+            require(value < 10 || value > 0)
+            field = value
+        }
+
+    /**
+     * Unit of concurrency. In combination with [maxMessages] determines the max number of messages
+     * that can be received every [sleepTime]. The upper bound of messages every [sleepTime] is
+     * [maxMessages] * [concurrency]
+     */
+    var concurrency: Int = 1
```

Comment:
Suggestion: I wouldn't prefer to limit the concurrency to 10 only, 100 would be better.

Reply:
we can increase it but im not sure if 100 is a good number. After a number of coroutines the sqs client does not benefit because it reaches an upper limit of concurrent requests.

Reply:
It would be the upper limit which is okay as then it is up to the user to handle that, but  I belive 10 is pretty low limiting